modules/ipi-install-bmc-addressing.adoc

```diff
@@ -8,6 +8,8 @@
 
 Most vendors support Baseboard Management Controller (BMC) addressing with the Intelligent Platform Management Interface (IPMI). IPMI does not encrypt communications. It is suitable for use within a data center over a secured or dedicated management network. Check with your vendor to see if they support Redfish network boot. Redfish delivers simple and secure management for converged, hybrid IT and the Software Defined Data Center (SDDC). Redfish is human readable and machine capable, and leverages common internet and web services standards to expose information directly to the modern tool chain. If your hardware does not support Redfish network boot, use IPMI.
 
+You can modify the BMC address during installation while the node is in the `Registering` state. If you need to modify the BMC address after the node leaves the `Registering` state, or if you need to modify the BMC credentials, you must disconnect the node from Ironic, edit the `BareMetalHost` resource, and reconnect the node to Ironic. See the _Additional resources_ section for details.
```

Comment:
I highly suspect that you do not need to detach the BMH for editing the BMC credentials.

Reply:
Yes, @MahnoorAsghar is correct. You can change the password any time.
To change the _address_ - effectively pointing the BMH at a completely different host - you must now disconnect from ironic first (previously this was not allowed at all).